apps/web/src/components/tx/DecodedTx/index.tsx

```diff
@@ -122,7 +94,7 @@ const DecodedTx = ({
                 </>
               )}
 
-              {txDetails && !showDecodedData && !isCreation ? (
```

Comment:
For other reviewers: this change confused me at first but the creation transaction renders fine.